spec/requests/atmosphere/api/virtual_machine_flavors_spec.rb

```diff
@@ -97,31 +97,31 @@
           flavors.each{|f| expect(f['memory']).to be >= required_mem }
         end
 
-        it "returns flavors with memory grater or equal to specified and available at given cs" do
-          get api("/virtual_machine_flavors?memory=#{required_mem}&compute_site_id=#{cs2.id}", user)
+        it "returns flavors with memory grater or equal to specified and available at given tenant" do
```

Comment:
Line is too long. [102/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.